reporting-tricks.md

```diff
@@ -40,4 +40,4 @@ To create an example API request compatible with Analysis Workspace:
 
   ![tips_response_body](/images/tips_response_body.png?raw=true)
 
-*Note: Before closing Analysis Workspace, you can turn off the debugger by opening your browser developer tools and pasting the following text into the console field: `adobe.tools.debug.includeOberonXml = false`. Press Enter and then refresh the page. The debug icons disappear once the debugger is off.*
+*Note: Before closing Analysis Workspace, you can turn off the debugger by clicking Disable Debugger from the Help menu.*
```

Comment:
@johnwight  - The original instructions were correct. You have to set the adobe.tools.debug.includeOberonXml field to false for the debug icons to disappear in the browser